examples/tutorials/api/maps.py

```diff
@@ -266,6 +276,156 @@
+# The `~gammapy.maps.TimeMapAxis.edges` method is shown which returns
+# `~astropy.units.Quantity` with respect to the reference time.
+# In this case the defined `~gammapy.maps.TimeMapAxis.edges_min` and `~gammapy.maps.TimeMapAxis.edges_max`.
+
+time_map_axis2 = TimeMapAxis(
+    edges_min=1 * u.d,
+    edges_max=11 * u.d,
+    reference_time=Time("2020-03-19"),
+)
+
+print(time_map_axis2.edges)
+
+######################################################################
+# The `~gammapy.maps.TimeMapAxis.time_edges` will return the `~astropy.time.Time` object directly
+
+print(time_map_axis2.time_edges)
+
+
+######################################################################
+# The `~gammapy.maps.TimeMapAxis.is_contiguous` method returns a boolean which indicates whether the
+# `~gammapy.maps.TimeMapAxis` is contiguous or not. This can be utilised on
+# the two previously defined `~gammapy.maps.TimeMapAxis` objects:
+
+print(time_map_axis.is_contiguous)
+
+print(time_map_axis2.is_contiguous)
+
+
+######################################################################
+# `~gammapy.maps.TimeMapAxis` also has both functionalities of `~gammapy.maps.TimeMapAxis.coord_to_pix`
+# and `~gammapy.maps.TimeMapAxis.coord_to_idx`.
+#
+# Start by choosing a time which we know is within the ``time_map_axis`` and see the results.
+
+time = Time(time_map_axis.time_max.mjd[0], format="mjd")
+
+print(time_map_axis.coord_to_pix(time))
+
+print(time_map_axis.coord_to_idx(time))
+
+
+######################################################################
+# This functionality can also be used with an array of `~astropy.time.Time` values.
+
+times = Time(time_map_axis.time_max.mjd, format="mjd")
+
+print(time_map_axis.coord_to_pix(times))
+
+print(time_map_axis.coord_to_idx(times))
+
+######################################################################
+# Note here we take a `~astropy.time.Time` which is outside the edges. For the
+# `~gammapy.maps.TimeMapAxis.coord_to_pix` method this will return ``nan`` and
+# for the `~gammapy.maps.TimeMapAxis.coord_to_idx` method it returns -1.
+
+print(time_map_axis.coord_to_pix(Time(time.mjd + 1, format="mjd")))
+
+print(time_map_axis.coord_to_idx(Time(time.mjd + 1, format="mjd")))
+
+
+######################################################################
+# The `~gammapy.maps.LabelMapAxis` object allows for handling of labels for map axes.
+# It provides an axis for non-numeric entries.
+#
+
+label_axis = LabelMapAxis(
+    labels=["dataset-1", "dataset-2", "dataset-3"], name="dataset"
+)
+
+print(label_axis)
+
+######################################################################
+# The labels can be checked using the `~gammapy.maps.LabelMapAxis.center` attribute:
+print(label_axis.center)
+
+
+######################################################################
+# To obtain the position of the label, one can utilise the `~gammapy.maps.LabelMapAxis.coord_to_pix` attribute
+
+print(label_axis.coord_to_pix(["dataset-3"]))
+
```

Comment:
Maybe you can plot on a Label axis to make the functionality clear

Reply:
Can you clarify what you mean?